resources/scss/componets/01_basics.scss

```diff
@@ -0,0 +1,173 @@
+/*Rubic font*/
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    line-height: 1.3;
+    font-weight: 400;
+    font-size: 14px;
+    font-family: 'Rubik', sans-serif;
+}
+
+
+
+
+/*color variables start*/
+
+$text:#eee;
+$white:#fff;
+$black:#000;
+$black1:#111;
+$theme:#F51963;
+$grayish:#7f7f7f;
+$yellow:#FBAE20;
+$light-bg:#f7f7f7;
+$gray-2: #ccc;
+$gray-3: #bbb;
+$gray-4: #777;
+$gray-5: #999;
+$gray-6: #575454;
+$news-bg: #FAF9FF;
+$dark: #222;
+
+/*color variables end*/
+
+
+
+.mt-30 {
+    margin-top: 30px;
+}
+
+.mb-30 {
+    margin-bottom: 30px;
+}
+
+.mb-80 {
+    margin-bottom: 80px;
+}
+
+.sec-wrap {
+    padding: 120px 0;
+}
+
+.sm-title {
+    font-size: 13px;
+    color: $theme;
+    letter-spacing: 1px;
+    font-weight: 400;
+    margin-bottom: 5px;
+}
```

Comment:
This is should be in **typography.scss**

Reply:
Fixed